packages/api-client/src/team/member/MemberAPI.ts

```diff
@@ -126,4 +133,39 @@ export class MemberAPI {
     const response = await this.client.sendJSON<Members>(config);
     return response.data;
   }
+
+  public async getMemberListCSV(
+    teamId: string,
+    progressCallback?: ProgressCallback,
+  ): Promise<RequestCancelable<MemberCSVResponse>> {
+    const cancelSource = axios.CancelToken.source();
```

Comment:
```suggestion
    const cancelSource = CancelToken.source();
```

Reply:
`'CancelToken' only refers to a type, but is being used as a value here.ts(2693)`